rules/false_positives/rancher.yara

```diff
@@ -3,11 +3,11 @@ rule pull_scripts : override {
     curl_chmod_relative_run_tiny = "medium"
     description = "pull-scripts"
   strings:
-    $binary = "BINARY_NAME=\"charts-build-scripts_${OS}_${ARCH}.exe\""
+    $binary_name = "BINARY_NAME=charts-build-scripts"
```

Comment:
I think this $binary_name rule should be sufficient - it's a superset of $charts.